test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -748,24 +722,23 @@ class test_ucp_peer_failure_rndv_abort : public test_ucp_peer_failure {
         auto *req   = ucs_container_of(uct_req, ucp_request_t, send.uct);
         auto stage  = req->send.proto_stage;
         auto *proto = req->send.proto_config->proto;
+        ucs::mock mock;
+
+        ucs_debug("progress_wrapper %s stage %d", proto->name, (int)stage);
```

Comment:
is it really needed? Typically we do not add debug prints to tests

Reply:
not really, removing